### PR TITLE
Update README for 22.05 release

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,5 +30,372 @@
...
-towards the next release. This branch is not usable during stabilization.**
+Triton Inference Server provides a cloud and edge inferencing solution
+optimized for both CPUs and GPUs. Triton supports an HTTP/REST and
+GRPC protocol that allows remote clients to request inferencing for
+any model being managed by the server. For edge deployments, Triton is
+available as a shared library with a C API that allows the full
+functionality of Triton to be included directly in an
+application.
+
+The current release of the Triton Inference Server is 2.22.0 and
```
Comment (dzier) [inline README.md:41]: Remove this paragraph for Readme's in release branches

File: README.md [hunk 2]
```diff
@@ -30,5 +30,372 @@
...
+* A [Java API](docs/inference_protocols.md#java-api) is similar to C API and
+  allows Triton to be linked directly to your Java application.
+
+* [Metrics](docs/metrics.md) indicating GPU utilization, server
+  throughput, and server latency. The metrics are provided in
+  Prometheus data format.
+
+## Documentation
+
+**The master branch documentation tracks the upcoming,
```
Comment (dzier) [inline README.md:155]: Remove this paragraph for release branches

**Comment by dzier** [comment]
Misha, let's hold off on this update. I am currently working on an entirely new README for main that I also want to debut in 22.05. So all of this will be removed.

**Review by dzier — CHANGES_REQUESTED** [review]

**Review by GuanLuo — COMMENTED** [review]
Need to address @dzier 's comments, otherwise looks good to me.

File: README.md (hunk 1 above)
Comment (mc-nv) [reply README.md]: @dzier - I've removed mentioned paragraph

File: README.md (hunk 2 above)
Comment (mc-nv) [reply README.md]: @dzier - I've removed mentioned paragraph

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Release Notes for 2.22.0
+
+## New Freatures and Improvements
+
+Put here what we used to put in "What's New"
```
Comment (GuanLuo) [inline RELEASE.md:33]: Based on this comment, shouldn't you move the corresponding section from `docs/README.md` to here?

**Review by dzier — APPROVED** [review]

**Pull request merged**